velox/functions/prestosql/window/tests/RankTest.cpp

```diff
@@ -46,6 +48,8 @@ TEST_F(RankTest, singlePartition) {
   });
 
   testTwoColumnInput({vectors}, "rank");
+  testTwoColumnInput({vectors}, "dense_rank");
+  testTwoColumnInput({vectors}, "percent_rank");
 }
 
 TEST_F(RankTest, randomInput) {
```

Comment:
Please add a test for a case with only one row per partition for covering the new condition you added.

Reply:
Thanks for the suggestion, added test case for this. I feel it would be better if WindowTestBase is modified to test for different vector sizes (for all values less than a given size, maybe?) instead of setting the vector size in the unit test cases. Shall I do that or is this new test case enough?

Reply:
The vector size is needed for makeRowVector and that changes from test to test, so there could be repetition. But the current test is sufficient for now.

Reply:
Got it, thanks!